LibGAP.jl/src/initialization.jl

```diff
@@ -25,7 +25,7 @@ read_sysinfo_gap = function(dir::String)
 end
 
 function error_handler(message)
-    print(message)
+    error("Error thrown by GAP")
```

Comment:
Could this perhaps include the message?

Reply:
I will check how to achieve this. It seemed to me that the message gets printed by GAP anyway.

Reply:
Oh, I see: I thought this is just a mater of printing `message`. But now looking at how the "error callback" is called, I realized that it actually gets no arguments -- and this signature here is incorrect.